demoapp/src/main/java/me/uport/sdk/demoapp/MainListActivity.kt

```diff
@@ -6,12 +6,16 @@ import android.support.v7.app.AppCompatActivity
 import android.widget.ArrayAdapter
 import android.widget.Toast
 import kotlinx.android.synthetic.main.simple_list.*
+import me.uport.sdk.demoapp.key_protection.KeyProtectionListActivity
+import me.uport.sdk.demoapp.managing_jwt.SignJWTListActivity
+import me.uport.sdk.demoapp.managing_jwt.VerifyJWTActivity
+import me.uport.sdk.demoapp.request_flows.RequestFlowListActivity
 
 /**
  *
- * Main screen for the demo app
- * This lists all the features to be demonstrated within the app
- * Clicking on an item opens up a new activity for the specific feature
+ * Main request flow screen for the demo app
```

Comment:
This kdoc comment block makes more sense in the `RequestFlowListActivity.kt` and the one from there should be here. It's probably a copy paste error.

Reply:
fixed